.circleci/config.yml

```diff
@@ -32,3 +32,8 @@ jobs:
             export PATH=$PATH:$HOME/.local/bin
             coverage run --source ffc -m pytest -s test/
             coveralls
+      - run:
+          name: Rerun tests (check caching)
+          command: |
+            export PATH=$PATH:$HOME/.local/bin
```

Comment:
Why is the path addition required?

Reply:
Don't know - I just copied from previous invocation above.

Reply:
Try removing - shouldn't be needed in this case. The case above calls some binaries in `$HOME/.local/bin` but this case doesn't.